src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,87 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number[],
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const paginationNumbs = Math.ceil(total[total.length - 1] / perPage);
+  const sliseFrom = (currentPage - 1) * perPage;
+  const sliseTo = perPage * currentPage;
+
+  return (
+    <>
```

Comment:
this wrapper is redundant
```suggestion
```

Reply:
Sorry, why redundant, if i have two tag 'ul'

Reply:
sorry, my mistake. Haven't noticed you have two `ul`, thought it was only one